greenkeeper.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
@marcelomorgado Do you foresee any problems with this greenkeeper.json file?

Reply:
No, not really.

Reply:
Okay, thanks. Resolving.